packages/react-email/src/utils/static-node-modules-for-vm.ts

```diff
@@ -89,4 +91,5 @@ export const staticNodeModulesForVM = {
   'node:buffer': buffer,
   assert,
   'node:assert': assert,
+  'async_hooks': asyncHooks,
```

Comment:
Could you just add in the `node:` prefixed variant here?

Reply:
9a559e85d72d4c4de835a272460a0de232182858